### DISCUSSION TITLE
Can't compile Windows - C++ 20

### DISCUSSION THREAD

**Discussion by pierricgimmig** [google/orbit#4446] [Q&A] [answered]
```
C:\git\orbit\src\ConfigWidgets\PropertyConfigWidgetTest.cpp(17,8): error C7555: use of designated initializers requires at least '/s
td:c++20' [C:\git\orbit\build_msvc2019_relwithdebinfo\src\ConfigWidgets\ConfigWidgetsTests.vcxproj]
C:\git\orbit\src\ConfigWidgets\PropertyConfigWidgetTest.cpp(29,8): error C7555: use of designated initializers requires at least '/s
td:c++20' [C:\git\orbit\build_msvc2019_relwithdebinfo\src\ConfigWidgets\ConfigWidgetsTests.vcxproj]
```

@beckerhe , is the CI not building Windows anymore? The Windows build is impossible to compile due to many sources of error.

**Comment by florian-kuebler** [comment, marked as answer]
Hey @pierricgimmig, we took a big step to massively simplify our build and build infrastructure, which allowed us to get rid of all the internal build jobs. This will be make all our lives much easier in the future and allows hopefully a low maintenance overhead. But we need to get there first.
 
As for the limited resources, we started this all out with support for Ubuntu 22.04 first. The Github CI has also perfect support for Windows builds. So as soon as we are back to a working Windows build (which we are looking into), we are going to set up a CI job AND automated nightly release for Windows, too. But note, that Windows support will very likely be a low-priority maintenance mode, as soon as we have it back working.

We can't make any promises on supporting Ubuntu 20.04 or lower. Contributions are very welcome, though, and we can for sure help out setting up CI/CD for this, if needed.

**Comment by pierricgimmig** [comment]
Ok that's great news, thanks a lot for the detailed answer @florian-kuebler , and thanks @beckerhe for all the dependency changes that will make for a simpler build process. Cheers!
